tox.ini

```diff
@@ -1,17 +1,23 @@
 [tox]
-envlist = py27,py34,py35,py36,py37,cov-report,cov-store
+envlist = flake8,py27,py34,py35,py36,py37,cov-report,cov-store
 
 [testenv]
 use_develop = true
 deps =
-    py27: mock==2.0.0
+    py27: mock>=2.0.0
     -r{toxinidir}/requirements-test.txt
 setenv =
     COVERAGE_FILE = tests/.coverage.{envname}
     PYTHONDONTWRITEBYTECODE=1
 commands =
     coverage run -m pytest {toxinidir}/tests {toxinidir}/flaskbb {posargs}
 
+[testenv:flake8]
```

Comment:
Might add a more intense task that runs other tools such as black, flake8-bugbear, etc. Travis has an option to allow specific jobs to fail so we could even run it there.